### ISSUE TITLE
Parts of app are not drawn until window is manually resized

### ISSUE THREAD

**Issue by koodimetsa** [fyne-io/fyne#495]
**Describe the bug**
When the app starts, parts of the content is missing. When the app's window is manually resized, all of the content starts showing.

**To Reproduce**
Here is sample code for the issue.

```
import (
	"fyne.io/fyne/layout"
	"fyne.io/fyne/widget"

	"fyne.io/fyne"
	"fyne.io/fyne/app"
)

func main() {
	a := app.New()

	w := a.NewWindow("NHL")
	w.Resize(fyne.NewSize(1000, 600))

	container := fyne.NewContainerWithLayout(layout.NewHBoxLayout())

	container.AddObject(NewTeamsWidget())

	w.SetContent(
		container,
	)

	w.ShowAndRun()
}

//NewTeamsWidget creates widget for selecting team
func NewTeamsWidget() fyne.CanvasObject {
	vBox := widget.NewVBox()

	scroller := widget.NewScrollContainer(vBox)
	teams := []string{"Anaheim Ducks", "Boston Bruins", "Buffalo Sabres", "Calgary Flames", "Carolina Hurricanes", "Chicago Blackhawks", "Colorado Avalanche", "Columbus Blue Jackets", "Dallas Stars", "Detroit Red Wings"}
	for index := 0; index < len(teams); index++ {
		t := teams[index]
		vBox.Append(widget.NewCheck(t, func(checked bool) {

		}))
	}

	return scroller
}

```

**Device (please complete the following information):**
 - OS: macOS
 - Version: Catalina 10.15
 - Go version: 1.13
 - Fyne version: 1.1.2

**Comment by koodimetsa** [comment]
Resize from the code after items are added also makes the content to be drawn fully.
E.g.
`w.Resize(fyne.NewSize(1000, 601))`

**Comment by andydotxyz** [comment]
I cannot replicate this on my Linux bug so looks like macOS specific - I will try to replicate on my office mac tomorrow

**Comment by andydotxyz** [comment]
Confirmed. This is on both master and develop but happens only on macOS.
It seems to be specific to a scrollcontainer being placed in a window that resizes before being shown.
The following workaround should help until we track down the bug:

```go
        w.Show()
        w.Resize(fyne.NewSize(1000, 600))
        a.Run()
```

**Comment by andydotxyz** [comment]
This should now be resolved on the HEAD of develop - ready for 1.2 release :)
